code/modules/mob/living/carbon/xenomorph/embryo.dm

```diff
@@ -229,3 +233,13 @@
 	if(species.species_flags & NO_PAIN)
 		return
 	emote("burstscream")
+
+/obj/item/alien_embryo/proc/adjust_boost_timer(amount, capped = 0, override_time = FALSE)
```

Comment:
autodoc

Reply:
Done?

Reply:
The format isn't correct. If you want multiline autodoc, go:
/**
	Helloworld
	anotherline
	* sometimes dot points to explain args or whatever
	*  blahblah

*/

That said, I think this is a pretty self explainatory proc, and you probably don't need to explain the args.